draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -975,21 +975,34 @@ the SD-JWT.
 
 ## Selectively-Disclosable Validity Claims {#sd-validity-claims}
 
-Claims controlling the validity of the SD-JWT, such as `nbf`, `iat`, and `exp`,
-are usually included in plaintext in the SD-JWT payload, but MAY be
-selectively disclosable instead. In this case, however, it is up to the Holder
-to release the claims to the Verifier. A malicious Holder may try to hide, for
+An Issuer MUST NOT allow any security-critical content to be selectively disclosable.
```

Comment:
Maybe we should be more specific here:
```suggestion
An Issuer MUST NOT allow any content to be selectively disclosable that is critical for evaluating the
SD-JWT's authenticity or validity.
```
(Not sure if `authenticity or validity` captures all things we might want to say here.)

Reply:
I think it probably captures enough.